test/integration/apiserver/apply_test.go

```diff
@@ -20,24 +20,43 @@ import (
 	"testing"
 
 	"k8s.io/apimachinery/pkg/types"
+	genericfeatures "k8s.io/apiserver/pkg/features"
+	utilfeature "k8s.io/apiserver/pkg/util/feature"
 )
 
+// TestApplyAlsoCreates makes sure that PATCH requests with the apply content type
+// will create the object if it doesn't already exist
 func TestApplyAlsoCreates(t *testing.T) {
+	if err := utilfeature.DefaultFeatureGate.Set(string(genericfeatures.ServerSideApply) + "=true"); err != nil {
```

Comment:
TODO: make setup do this optionally, and make it temporary rather than affecting global state.

Reply:
I'm pretty sure I just copied this pattern from another integration test.
https://github.com/kubernetes/kubernetes/blob/b3837d004adab8e1f5f0eae7fdd2ddcd614258a0/test/integration/apiserver/apiserver_test.go#L173-L175
I'll add the TODO to this PR, but should I file an issue about that one too?